pkg/virt-config/config-map.go

```diff
@@ -70,18 +72,20 @@ type ConfigModifiedFn func()
 func NewClusterConfig(configMapInformer cache.SharedIndexInformer,
 	crdInformer cache.SharedIndexInformer,
 	kubeVirtInformer cache.SharedIndexInformer,
+	hostDevConfigMapInformer cache.SharedIndexInformer,
```

Comment:
are we sure we need a separate `kubevirt-host-device-plugin-config` configmap?

By exposing the configuration of this permitted host device list as part of the KubeVirt CRD, we're giving a well defined/structured API for setting these values. Why would we want people to use a config map?

Reply:
There are some advantages from my point of view. It keeps the permitted devices separately from all other kubevirt configurations.
It keeps a list of permitted devices dynamic, so the admin can add/remove devices and virt-handler will start/stop device plugins for added/removed devices.

If I understand the CRD changes correctly, every change will lead to a restart of the virt-handler. Maybe it's fine...

Reply:
Removed. Now we are just whitelisting the permitted host devices to KubeVirt CRD